test/cppsuite/test_harness/runtime_monitor.h

```diff
@@ -109,13 +110,96 @@ class cache_limit_statistic : public statistic {
     int64_t limit;
 };
 
+static std::string
+collection_name_to_file_name(const std::string &collection_name)
+{
+    /* Strip out the URI prefix. */
+    const size_t colon_pos = collection_name.find(':');
+    const auto stripped_name =
+      (colon_pos == std::string::npos) ? collection_name : collection_name.substr(colon_pos + 1);
```

Comment:
If there is no colon lets testutil_die for the time being.

Reply:
14f602f